### ISSUE TITLE
Website connecting to GitHub API

### ISSUE THREAD

**Issue by willosof** [bitfocus/companion#560]
We need a website that can show us all open module pull requests, bugs, etc. It's hard to follow 100 repos. (And no, I don't regret regret splitting modules out in submodules. it's the best!)

**Comment by mfoulks3200** [comment]
Working on this now, are there any must haves? 
Currently I'll be tracking:

- Repository Activity (general activity)
- Pull Requests
- Issues
- Commits

**Comment by mfoulks3200** [comment]
Preliminary UI
![alt text](https://i.imgur.com/VfWDkUZ.png)

**Comment by willosof** [comment]
We also want to know the submodule position versus the companion repository (how many commits behind the modules are from the master)

**Comment by willosof** [comment]
* only repos matching companion-module-* 
* it should report if any repo matching that is NOT linked into https://github.com/bitfocus/companion/tree/master/lib/module
* make a compare log (commit message log with link to the commit itself) against the version linked from https://github.com/bitfocus/companion/tree/master/lib/module to the master of the module, that way we see what we've not merged into master :)

**Comment by willosof** [comment]
Compare like this:
```
https://github.com/bitfocus/companion-module-atem/compare/3fd46a0...master
```

Here we take the current atem @ 3fd46a0 in companion/lib/modules/ and compare it to companion-module-atem master :) giving us a nice overview of what's being done and what we're merging in when we upgrade a module in core.
